db/compaction_picker.cc

```diff
@@ -1289,8 +1289,16 @@ uint32_t LevelCompactionBuilder::GetPathId(
         return p;
       } else {
         current_path_size -= level_size;
-        level_size = static_cast<uint64_t>(
-            level_size * mutable_cf_options.max_bytes_for_level_multiplier);
+	if (ioptions.level_compaction_dynamic_level_bytes) {
```

Comment:
can you fix the whitespace? spaces instead of tabs, indents are two spaces, line length limit is 80 chars